packages/did-provider-oyd/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@ownyourdata/did-provider-oyd",
```

Comment:
make sure to change this to `@veramo`

Reply:
changed all relevant places to `@veramo`